<launch>
	<!-- Launch kinematics test -->
	<include file="$(find urGovernor)/launch/kinematicsTest.launch"></include>

	<!-- Launch deltaTest node -->
	<node pkg="urGovernor" type="deltaTest" name="deltaTest" output="screen">
		<rosparam command="load" file="$(find urVision)/config/common.yaml" />
		<rosparam command="load" file="$(find urGovernor)/config/governor.yaml" />
	</node>
</launch>
